main/lsp/tools/make_lsp_types.cc

```diff
@@ -1304,12 +1304,6 @@ void makeLSPTypes(vector<shared_ptr<JSONClassType>> &enumTypes, vector<shared_pt
                                             makeField("message", JSONString),
                                         },
                                         classTypes);
-    auto SorbetFenceParams = makeObject("SorbetFenceParams",
-                                        {
-                                            makeField("id", JSONInt),
-                                            makeField("advanceIfStaleDataAvailable", JSONBool),
-                                        },
-                                        classTypes);
```

Comment:
I kind of want to keep the stronger type of `SorbetFenceParams` here and just delete `advanceIfStaleDataAvailable`, but up to you.

Reply:
Some of the changes I made by simply reverting old PRs that still applied cleanly, this was one of them. But I can put this one back.

Reply:
This is blocking the stack though, so I'll do that in a future change.